drf_recaptcha/validators.py

```diff
@@ -18,8 +22,31 @@ class ReCaptchaValidator:
         "captcha_invalid": "Error verifying reCAPTCHA, please try again.",
         "captcha_error": "Error verifying reCAPTCHA, please try again.",
     }
-    recaptcha_client_ip = ""
-    recaptcha_secret_key = ""
+    default_recaptcha_client_ip = ""
+    default_recaptcha_secret_key = ""
+
+    def __call__(self, value, serializer_field=None):
```

Comment:
```suggestion
    def __call__(self, value, serializer_field):
```

rest_framework.fields.Field.run_validators

```python
    def run_validators(self, value):
        """
        Test the given value against all the validators on the field,
        and either raise a `ValidationError` or simply return.
        """
        errors = []
        for validator in self.validators:
            try:
                if getattr(validator, 'requires_context', False):
                    validator(value, self)  # <<<---- we always send
                else:
                    validator(value)
            except ValidationError as exc:
```

Reply:
Please run tests on your branch for all versions. Probably I added it for some old drf version. If this change break only the oldest let's drop support of this version.
I will increase major version anyway

Reply:
I think you've added this because it was easier to test, like: https://github.com/llybin/drf-recaptcha/blob/7ad060dca82fa470512f94ba48cf3b39a9bc657d/tests/test_testing.py#L14 and https://github.com/llybin/drf-recaptcha/blob/7ad060dca82fa470512f94ba48cf3b39a9bc657d/tests/test_testing.py#L31 

i think it's okey to ignore it for now

Reply:
looks like as shit-code if I did it only for that :)

https://www.django-rest-framework.org/community/3.11-announcement/#validator-default-context

the minimal version is 3.1, https://github.com/llybin/drf-recaptcha/commit/d69bbb7a6e719320d5ffe85138ab8c1fa8a23b2f so remove it yep